plugin/core/open.py

```diff
@@ -15,6 +15,20 @@
 opening_files = {}  # type: Dict[str, Tuple[Promise[Optional[sublime.View]], ResolveFunc[Optional[sublime.View]]]]
 
 
+def _return_existing_view(flags: int, is_view_in_active_group: bool, specified_group: int) -> bool:
+    # the specified_group is supplied
+    if specified_group > -1:
+        return is_view_in_active_group
```

Comment:
I wonder if this still isn't a logic error here? I think if a group is given, then it should return whether the view is in the specified group, and not whether it's in the active group, no?

Reply:
Thanks for pointing this out again @jwortmann, i got myself confused.